UploadFile/Table/Tab50410.BCPUpload.al

```diff
@@ -0,0 +1,56 @@
+table 50410 "BCP Upload"
+{
+    DataClassification = CustomerContent;
+    InherentPermissions = RIMDX;
```

Comment:
Why you added it?

Reply:
No particular reason, i was just researching and found out about that property and decided to try it out and see. should i remove it? any implication ?

Reply:
There is no point in adding something to the code that doesn't improve user experience or solve some problem.

Reply:
Alright noted.